components/ScoreCard.vue

```diff
@@ -1163,7 +1195,7 @@ export default class extends Vue {
       }
 
       if (swResponse && swResponse.data) {
-        await this.scoreServiceWorker(cleanUrl, swResponse.data);
```

Comment:
Changes start here, stuff above this is linter updates

Reply:
K. Can you turn off linter auto-formatting? Makes code reviews harder.

Reply:
Yeah definitely. I thought I had but still had it running on VSCode on this machine.